pypiserver/config.py

```diff
@@ -382,6 +391,15 @@ def get_parser() -> argparse.ArgumentParser:
             "index."
         ),
     )
+    run_parser.add_argument(
+        "--health-endpoint",
+        default=DEFAULTS.HEALTH_ENDPOINT,
+        type=health_endpoint_arg,
+        help=(
+            "The liveness endpoint. Always response 200 OK, otherwise means"
+            "that the service is unhealthy or dead."
```

Comment:
```suggestion
            "Configure a custom liveness endpoint. It always returns 200 OK if the service is up." 
            "Otherwise it means that the service is unhealthy or dead."
```

I think this makes the purpose of this setting a little more clear :)

Reply:
The line length in this suggestion might need to be double checked 😁